File: IMT2021545_CPP_Lab5/TV.cpp
#include<iostream>
#include<bits/stdc++.h>
using namespace std;
#include"TV.h"

void TV::set_valtv(string br,string ty,int si,int pr,int ra)
{
  // setting all the relevant information
  type=ty;
  price=pr;
  brand=br;
  size=si;
  rating=ra;
}

// getters
string TV::getBrand()
{
  return brand;
}

int TV::getRating()
{
  return rating;
}

int TV::getSize()
{
  return size;
}

// method to print all the information
void TV::getInfo() {
    string inf = "TV " + getBrand() + " " + getType() + " " + to_string(getSize()) + " " + to_string(getPrice()) + " " +  to_string(getRating());
    cout << inf << endl;
}

File: IMT2021545_Lab5/TV.java
public class TV extends Product {

    private String brand;
    private int size;
    private int rating;

    //setter method
    void setVals(String b,String t,int s,int p,int r)
    {
        brand=b;
        rating=r;
        size=s;
        super.setVals(t,p);
    }

    //getter methods
    String getBrand(){
        return brand;
    }

    int getSize(){
        return size;
    }
    
    int getRating(){
        return rating;
    }
}
